src/policy/fees.cpp

```diff
@@ -901,7 +901,8 @@ bool CBlockPolicyEstimator::Write(CAutoFile& fileout) const
 {
     try {
         LOCK(m_cs_fee_estimator);
-        fileout << 149900; // version required to read: 0.14.99 or later
+        //ELEMENTS: we changed the buckets in 0.18.1.7
+        fileout << 180107; // version required to read: 0.14.99 or later
```

Comment:
Comment on this line needs updated?

Reply:
Hmm, I don't think so. I left it op purpose. That's the version to read the file structure that I wanted to leave intact. The new version requirement is for the bucket sizes. I can make that more explicit if you want.

Reply:
Hm, I think I don't really understand what's going on in this section of code, maybe we can chat about it.